docs/moz-addon-manager.md

```diff
@@ -0,0 +1,204 @@
+# Develop features for mozAddonManager
+
+The AMO and Discovery Pane apps use the
+[`mozAddonManager`](https://bugzilla.mozilla.org/show_bug.cgi?id=1310752)
+web API to achieve a more seamless add-on installation user experience.
+However, this API is only available to a limited list of domains.
+If you go to https://addons.mozilla.org (production) in Firefox then `mozAddonManager`
+is available on the page. If you go to a development site then it's not.
+
+## Turning on mozAddonManager for development
+
+To access `mozAddonManager` on a development site like
+https://addons-dev.allizom.org or https://addons.allizom.org
+go to `about:config` in Firefox and set this property to `true`:
```

Comment:
It's worth adding a note here that IIRC this will mess up all the existing add-on installs in your browser which really sucks, so use this option with caution.